Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/VREE.MSG

```diff
@@ -123,13 +123,12 @@
 
 {1000}{}{Master}
 {1001}{}{War}
-{1002}{}{Disks|Holodisk|Holodisks|Disk|Holo}
+{1002}{}{Disks|Holodisk|Holodisks|Disk|Holo|Holo-disk|Holo-disc|Holodisc}
 {1003}{}{Brotherhood|BoS|Brotherhood of Steel}
 {1004}{}{Water Chip|Waterchip|Chip}
 {1005}{}{Elder|High Elder}
 {1006}{}{Rhombus|Head Paladin}
-{1007}{}{Exodus}
-{1008}{}{Army|Strange Army|Mutant Army}
+{1008}{}{Army|Strange Army|Mutant Army|Missing Caravans|Caravans|Caravan|Missing Caravan}

```

Comment:
Removed the line about the Exodus.

Reply:
Exodus was removed because Vree speaks the same default message anyway, reduced redundancy.

Reply:
It's a different reaction than a non-valid keyword, though. Which means the game at least acknowledges the keyword.